api/java/org/openyolo/api/CredentialClient.java

```diff
@@ -332,6 +364,14 @@ public CredentialSaveResult getCredentialSaveResult(Intent resultData) {
         }
     }
 
+    /**
+     * Disables automatically signing a user until a successful response is given to
+     * {@link CredentialClient#getCredentialRetrieveResult(Intent)}.
+     */
+    public void disableAutoSignIn() {
```

Comment:
I thought of  ```requireUserMediation``` as provider facing terminology and ```disableAutoSignIn``` as client facing.  The inconsistency might be confusing, but I am not sure how many people will actually run into it in practice.